examples/github-repositories-custom-plugin/app.tsx

```diff
@@ -11,5 +11,6 @@ const gitHubReposPlugin = createGitHubReposPlugin({
 
 autocomplete({
   container: '#autocomplete',
+  debug: true,
```

Comment:
```suggestion
```

I think we can remove the `debug` option from the examples (except the playground)

Reply:
Woops!